README.md

```diff
@@ -65,12 +65,20 @@ All components receive configuration via environment variables. The following va
 | Variable | Default | Explanation |
 | -------- | ------- | ----------- |
 | `CASTELLUM_ASSET_MANAGERS` | *(required)* | A comma-separated list of all asset managers that can be enabled. This configures what kinds of assets Castellum can handle. See [`docs/asset-managers/`](./docs/asset-managers/) for which asset managers exist. |
-| `CASTELLUM_DB_URI` | *(required)* | A [libpq connection URI][pq-uri] that locates the Castellum database. The non-URI "connection string" format is not allowed; it must be a URI. |
+| `CASTELLUM_DB_USERNAME` | `postgres` | Username of the user that Castellum should use to connect to the database. |
+| `CASTELLUM_DB_PASSWORD` | *(optional)* | Password for the specified user. |
+| `CASTELLUM_DB_HOSTNAME` | `localhost` | Hostname of the database server. |
+| `CASTELLUM_DB_PORT` | `5432` | Port on which the PostgreSQL service is running on. |
+| `CASTELLUM_DB_NAME` | `castellum` | The name of the database. |
+| `CASTELLUM_DB_CONNECTION_OPTIONS` | `sslmode=disable` | Database connection options. |
```

Comment:
```suggestion
| `CASTELLUM_DB_CONNECTION_OPTIONS` | `` | Database connection options. |
```
We should not advertise no-TLS as a safe default. It's okay in our deployment because we're reaching the Postgres via an isolated network, but generally TLS is advised.